test/integration/scheduler/predicates_test.go

```diff
@@ -1074,14 +1075,40 @@ func TestUnschedulablePodBecomesSchedulable(t *testing.T) {
 			pod: &pausePodConfig{
 				Name: "pod-1",
 			},
-			update: func(cs kubernetes.Interface) error {
+			update: func(cs kubernetes.Interface, _ string) error {
 				taint := v1.Taint{Key: "test", Value: "test", Effect: v1.TaintEffectNoSchedule}
 				if err := testutils.RemoveTaintOffNode(cs, "node-tainted", taint); err != nil {
 					return fmt.Errorf("cannot remove taint off node: %v", err)
 				}
 				return nil
 			},
 		},
+		{
+			name: "pod gets deleted",
```

Comment:
other pod gets deleted.
Sorry I missed this before

Reply:
no worries! makes sense, I will add it